packs/data/feats.db/curse-maelstrom-dedication.json

```diff
@@ -13,7 +13,7 @@
+            "value": "<p>@UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.KRUWPDzuwG0LC26d]{Curse Maelstrom}</p>\n<p>A horrible curse has left a lingering stain on your soul, and its effects continue to plague you even if the curse's other effects have ended. You can choose to suppress it, but if you instead let it manifest, you can channel its power in your favor. During an encounter, when you succeed at a check, the GM can offer to have you roll again, taking the second result. This is a misfortune effect. If you do so, you enter a curse maelstrom state. If the misfortune effect is somehow canceled or negated (by a fortune effect, for example), you don't enter a curse maelstrom state.</p>\n<p>In addition, the curse is jealous of its place within your soul and gladly lashes out whenever others dare to usurp that position, granting you its power without the usual price. If a foe places a misfortune effect on your roll and the effect applies to your roll, you enter a curse maelstrom state, and if you fail a saving throw against a foe's curse effect and are affected by the curse, you also enter a curse maelstrom state. No matter the source, you can only enter a curse maelstrom state during an encounter, and if you don't end the state on your own, it ends at the end of the encounter. Once it ends, you can't enter a curse maelstrom state again for 1 minute.</p>\n<p>While in a curse maelstrom state, you can't benefit from fortune effects, and they also don't cancel misfortune effects on you; they simply have no effect. While in this state, all creatures other than you within a @Template[type:emanation|distance:10] take a -1 status penalty to all saving throws and skill checks due to the storm of bad luck swirling out of your body. You gain the Expel Maelstrom action, which you can use to focus the brunt of your curses onto an unlucky target.</p>"
```

Comment:
I don't think we should link at the start of the dedication, move it to after the description. I guess this archetype doesn't require you to take more than the dedication though, weird.